arangosh/Dump/DumpFeature.cpp

```diff
@@ -209,6 +209,28 @@ void flushWal(arangodb::httpclient::SimpleHttpClient& client) {
 }
 }  // namespace
 
+namespace {
+bool isIgnoredHiddenEnterpriseCollection(
```

Comment:
Can we move this helper method somewhere more general ?

Reply:
Sure, do you have a suggestion? Maybe static method on `LogicalCollection`?